VirtoCommerce.CustomerModule.Test/LogChangesMemberChangedEventHandlerTests.cs

```diff
@@ -0,0 +1,221 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using VirtoCommerce.CustomerModule.Data.Handlers;
+using VirtoCommerce.Domain.Commerce.Model;
+using VirtoCommerce.Domain.Common.Events;
+using VirtoCommerce.Domain.Customer.Events;
+using VirtoCommerce.Domain.Customer.Model;
+using VirtoCommerce.Platform.Core.ChangeLog;
+using VirtoCommerce.Platform.Core.Common;
+using Xunit;
+
+namespace VirtoCommerce.CustomerModule.Test
+{
+    public class LogChangesMemberChangedEventHandlerTests
+    {
+        [Fact]
```

Comment:
I would to suggest use Data-Driven approach for pass incoming parameters and for assertions. See as example https://github.com/VirtoCommerce/vc-module-core/blob/master/VirtoCommerce.CoreModule.Tests/AmountBasedRewardTests.cs#L14

Reply:
Here we need to verify that `changesLogService.SaveChanges()` method is called with correct operation logs, should I use `.Callback` with `Assert` instead?